### ISSUE TITLE
Can't train my own dataset

### ISSUE THREAD

**Issue by hupanlsj** [zzh8829/yolov3-tf2#114]
Question1:
    I have a mission to detect the boat ,just one class.The system is ubuntu 18.04 and tensorflow 2.0.0.I generate the tfrecord from the object detection API.Firstly,I use the fine-tune to train my dataset.And I config the parameter num_classes to 1 class.I get an error that means the dimension 18 is not compatible to the dimension 255 in the "load_weight".I realize is the class problem.If the numble classes is 80,the dimension will be the 255(3*(80+4+1)),the 18 means(3*(1+4+1)).Why get this problem,I analysis the problem is because the load weight(yolo3 weight) is for the 80 classes,not the class 1.I don't know if the code can fine-tune the model?Any body fine-tune the code right

Question 2:
I train my dataset without fine-tune,just train all the parameters. I get a model,but run the model for the detection.I get an error 
![image](https://user-images.githubusercontent.com/30136005/70018871-67f26900-15c2-11ea-86d7-940f8b7ccf03.png)Anyone meet this error before?

If you can answer my confusion，I  appreciate it.

**Comment by BahadirGLCK** [comment]
You can check this solution above the train.py nan values #103   issue.
Maybe this solution is solved your problem.

**Comment by hupanlsj** [comment]
@BahadirGLCK  Thanks for your reply.But  I think that solution can not solve my problem.Because my loss is not none.
![image](https://user-images.githubusercontent.com/30136005/70102292-a0ea1680-1672-11ea-83b8-0bc3ce837885.png)
And the cors in my data is just xmin,ymin,xmax,ymax.
I can get the model.But when I use the model to detect my picture.It didn't work and get an error.
![image](https://user-images.githubusercontent.com/30136005/70102378-d7c02c80-1672-11ea-81d4-34cadd7eb038.png)

**Comment by hupanlsj** [comment]
![image](https://user-images.githubusercontent.com/30136005/70103507-4b176d80-1676-11ea-981f-bc941f93d490.png)
Probably，I refused by the code"model.load_weight("yolov3.weight").I think the yolov3.weight is trained for the classes 80.If I transfer the model and load the weights firstly,we will get the incompatible error.Because the model we build is for the num_classes we set(mine is 1),but the yolov3 weight is design for class 80. 
If my idea is correct or wrong? If it's exist the problem in this repo,how can I modify this code.
